### PR TITLE
feat(const): added public constant library to share constants between projects

### PR DESCRIPTION
**Pull request by michaelmass**

### CODE REVIEW

**Review by chsir17 — APPROVED** [review]

File: const/src/prefixes.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
+export const prefixToObjectMap = {
+  act: 'action',
+  anly: 'analytics',
+  aud: 'audit',
+  bak: 'botApiKey',
+  bot: 'bot',
+  card: 'card',
+  cfg: 'configuration',
+  chn: 'channel',
+  cnv: 'conversation',
```
Comment (spg) [inline const/src/prefixes.ts:10]:
```suggestion
  conv: 'conversation',
```

File: const/src/prefixes.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
+export const prefixToObjectMap = {
+  act: 'action',
+  anly: 'analytics',
+  aud: 'audit',
+  bak: 'botApiKey',
+  bot: 'bot',
+  card: 'card',
+  cfg: 'configuration',
+  chn: 'channel',
```
Comment (spg) [inline const/src/prefixes.ts:9]:
```suggestion
  chan: 'channel',
```

File: const/src/prefixes.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
+export const prefixToObjectMap = {
+  act: 'action',
+  anly: 'analytics',
```
Comment (spg) [inline const/src/prefixes.ts:3]:
```suggestion
  analyt: 'analytics',
```

File: const/src/prefixes.ts [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  cfg: 'configuration',
+  chn: 'channel',
+  cnv: 'conversation',
+  evt: 'event',
+  file: 'file',
+  flow: 'workflow',
+  iak: 'integrationApiKey',
+  int: 'integration',
+  kb: 'knowledgeBase',
+  lmt: 'limit',
```
Comment (spg) [inline const/src/prefixes.ts:17]:
```suggestion
  lim: 'limit',
```

File: const/src/prefixes.ts [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  med: 'media',
+  msg: 'message',
+  node: 'node',
+  pat: 'personalAccessToken',
+  qta: 'quota',
```
Comment (spg) [inline const/src/prefixes.ts:22]:
```suggestion
  quota: 'quota',
```

File: const/src/prefixes.ts [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+  revt: 'recurringEvent',
+  sbx: 'sandbox',
+  sch: 'schema',
```
Comment (spg) [inline const/src/prefixes.ts:25]:
```suggestion
  schema: 'schema',
```

File: const/src/prefixes.ts [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+  ste: 'state',
+  tag: 'tag',
+  tbl: 'table',
+  user: 'user',
+  usg: 'usage',
+  wbh: 'webhook',
```
Comment (spg) [inline const/src/prefixes.ts:31]:
```suggestion
  webhk: 'webhook',
```

File: const/src/prefixes.ts [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+  wks: 'workspace'
```
Comment (spg) [inline const/src/prefixes.ts:32]:
```suggestion
  worksp: 'workspace'
```

File: const/src/prefixes.ts [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+  aud: 'audit',
+  bak: 'botApiKey',
+  bot: 'bot',
+  card: 'card',
+  cfg: 'configuration',
+  chn: 'channel',
+  cnv: 'conversation',
+  evt: 'event',
+  file: 'file',
+  flow: 'workflow',
```
Comment (spg) [inline const/src/prefixes.ts:13]:
```suggestion
  wflow: 'workflow',
```

File: const/src/prefixes.ts [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+  iak: 'integrationApiKey',
+  int: 'integration',
+  kb: 'knowledgeBase',
+  lmt: 'limit',
+  med: 'media',
```
Comment (spg) [inline const/src/prefixes.ts:18]:
```suggestion
  media: 'media',
```

File: const/src/prefixes.ts [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+  bot: 'bot',
+  card: 'card',
+  cfg: 'configuration',
+  chn: 'channel',
+  cnv: 'conversation',
+  evt: 'event',
+  file: 'file',
+  flow: 'workflow',
+  iak: 'integrationApiKey',
+  int: 'integration',
```
Comment (spg) [inline const/src/prefixes.ts:15]:
```suggestion
  integ: 'integration',
```

File: const/src/prefixes.ts [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+  act: 'action',
+  anly: 'analytics',
+  aud: 'audit',
+  bak: 'botApiKey',
+  bot: 'bot',
+  card: 'card',
+  cfg: 'configuration',
+  chn: 'channel',
+  cnv: 'conversation',
+  evt: 'event',
```
Comment (spg) [inline const/src/prefixes.ts:11]:
```suggestion
  event: 'event',
```

File: const/src/prefixes.ts [hunk 13]
```diff
@@ -0,0 +1,74 @@
+export const prefixToObjectMap = {
+  act: 'action',
+  anly: 'analytics',
+  aud: 'audit',
```
Comment (spg) [inline const/src/prefixes.ts:4]:
```suggestion
  audit: 'audit',
```

File: const/src/prefixes.ts [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+  iak: 'integrationApiKey',
+  int: 'integration',
+  kb: 'knowledgeBase',
+  lmt: 'limit',
+  med: 'media',
+  msg: 'message',
+  node: 'node',
+  pat: 'personalAccessToken',
+  qta: 'quota',
+  revt: 'recurringEvent',
```
Comment (spg) [inline const/src/prefixes.ts:23]:
```suggestion
  recevent: 'recurringEvent',
```

File: const/src/prefixes.ts [hunk 15]
```diff
@@ -0,0 +1,74 @@
...
+  sbx: 'sandbox',
+  sch: 'schema',
+  ste: 'state',
```
Comment (spg) [inline const/src/prefixes.ts:26]:
```suggestion
  state: 'state',
```

File: const/src/prefixes.ts [hunk 16]
```diff
@@ -0,0 +1,74 @@
...
+  tag: 'tag',
+  tbl: 'table',
+  user: 'user',
+  usg: 'usage',
```
Comment (spg) [inline const/src/prefixes.ts:30]:
```suggestion
  usage: 'usage',
```

File: const/src/prefixes.ts [hunk 17]
```diff
@@ -0,0 +1,74 @@
...
+  msg: 'message',
+  node: 'node',
+  pat: 'personalAccessToken',
+  qta: 'quota',
+  revt: 'recurringEvent',
+  sbx: 'sandbox',
+  sch: 'schema',
+  ste: 'state',
+  tag: 'tag',
+  tbl: 'table',
```
Comment (spg) [inline const/src/prefixes.ts:28]:
```suggestion
  table: 'table',
```

**Comment by spg** [comment]
See https://gist.github.com/fnky/76f533366f75cf75802c8052b577e2a5 for inspiration

**Comment by michaelmass** [comment]
We should keep a maximum of 6 characters for all prefixes.

**Pull request merged**